about.html

```diff
@@ -0,0 +1,152 @@
+            <p>info@porabo.live</p>
+          </div>
+        </div>
+    </main>
+    <section class="abt-logoContainer flex">
+      <div class="abt-logoTitle">
+        <h1 class="abt-logoHeading">Creative Common Global Summit 2015 Logo</h1>
+      </div>
+      <div class="abt-hrDiv flex">
+        <hr>
+      </div>
+      <div class="abt-logoHistory">
+        <p class="abt-paraGraph">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Suscipit inventore in sunt nemo dicta. Eveniet eum reprehenderit ipsa id quisquam dicta quis! Voluptatum aliquam quas repellat, vel atque hic temporibus? Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corporis voluptates est possimus recusandae optio odio assumenda? Maiores, in recusandae. Sapiente architecto veniam quia aspernatur tempore magnam eius atque quaerat iure?</p>
+      </div>
+      <div class="abt-logoImage">
+        <img src="assets/logo_Porabo_live.jpg" alt="logo of the company">
+      </div>
+      
+    </section>
+    <section class="abt-pastEvents flex">
+      <div class="abt-deviderHr flex">
+        <hr>
+      </div>
+      <div class="abt-pastEventTitle">
+        <h1 class="abt-logoHeading">See the past CC Global Summits</h1>
+      </div>
+      <div class="abt-hrDiv flex">
+        <hr>
+      </div>
+      <div class="abt-logoHistory">
+        <p class="abt-paraGraph">Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+      </div>
+      <div class="abt-pastEventpic flex">
+        <div class="confarence-1">
+          <img src="assets/Confarence-1.jpg" alt="Conference Picture">
+          <p>Confarence 2011</p>
+        </div>
+        <div class="confarence-1">
+          <img src="assets/Confarence-2.jpg" alt="Conference Picture">
+          <p>Confarence 2013</p>
+        </div>
+    </section>
+    <section class="abt-paartners">
+      <div class="textAndHr flex">
+        <h3 class="partnerText">Partners</h3>
+        <hr>
+      </div>
+      <div class="partnersUldiv flex">
+        <ul>
+          <li><img src="assets/Mozila_partners.svg" alt="Mozilla logo"></li>
+          <li><img src="assets/Google_partners.jpg" alt="Mozilla logo"></li>
+          <li><img src="assets/airbnb_partners.jpg" alt="Mozilla logo"></li>
+          <li><img src="assets/daumkakao_partners.jpg" alt="Mozilla logo"></li>
+          <li><img src="assets/naver_partners.jpg" alt="Mozilla logo"></li>
+        </ul>
+      </div>
+      
+    </section>
+    <footer class="footerDiv flex">
+      <div class="footerlogo flex">
+        <img src="assets/logo_Porabo_live.jpg" alt="Logo of the company">
+      </div>
+      <div class="addressAndCredit">
+        <address> 2015 Creative Commons Korea, Some Rights Reserved</address>
+        <p>Developed by <em>Nahid Raihan</em></p>
+        <p>Template credit <a class="no-decoration" href="https://www.behance.net/adagio07"><em>Cindy Shin</em></a></p>
+      </div>
+    </footer>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please pay attention to the differences in the images I've highlighted below and correct them`!` Change the `background color` of this `section` and `footer` to the correct color and give the required `filter` and `style` to the `images` in the section and `adjust` the position of the `elements` 🙏🏼 

### Desktop
![image](https://user-images.githubusercontent.com/63915024/233162681-feb67a00-1042-4899-a537-de97358ee116.png)

### Mobile
![image](https://user-images.githubusercontent.com/63915024/233162728-8aa0e323-9b55-4284-8d9e-f6804d67a441.png)

Reply:
@mukhammadrizooff 
Resolved